website/content/docs/commands/tls/cert.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: 'Commands: TLS Cert Create'
+---
+
+# Nomad TLS Cert Create
+
+Command: `nomad tls cert create`
```

Comment:
There's a format that all the command docs share, so copy those:

```suggestion
# Command: nomad tls cert create
```

(Applies to all the docs)